app/assets/stylesheets/core/_link.scss

```diff
@@ -3,13 +3,14 @@
   color: $govuk-error-colour;
 
   &:link,
-  &:visited {
+  &:visited,
```

Comment:
This looks like we're trying to show the error colour less often?

Reply:
I would say 'more often'. We now show the same shade for `:link`, `:visited`, `:hover` and `:active`, the only state when it's different is `:focus`

Reply:
Oh yeah - sorry, I was misreading the '&'